### PR TITLE
React app for comments

### PR DESCRIPTION
**Pull request by AnneTee**
## Summary of changes

Replaces our current comment functionality with a React app!

## To test

I've tested this locally one last time (after rebasing on source).

### Pull request reviewer

As the reviewer, I have verified the following:

- [x] I have tested the PR locally and/or in a staging environment

### CODE REVIEW

**Review by kostajh — APPROVED** [review]
Looks good! I leave it to you if you want to rewrite my commit messages or squash this into a single commit.

**Comment by AnneTee** [comment]
Thanks @kostajh, commits squashed, I'll merge when tests pass!

**Pull request merged**
